pandas/tseries/resample.py

```diff
@@ -319,9 +319,19 @@ def aggregate(self, arg, *args, **kwargs):
         self._set_binner()
         result, how = self._aggregate(arg, *args, **kwargs)
         if result is None:
-            return self._groupby_and_aggregate(arg,
-                                               *args,
-                                               **kwargs)
+            # It seems there is an ambiguity in having both named args
+            # and variadic in the same function, so we need to distinguish here,
+            # otherwise the first element of args is "eaten" by grouper in
+            # _groupby_and_aggregate.
+            if len(args) == 0:
+                return self._groupby_and_aggregate(arg,
```

Comment:
no you need to figure out the reason and not put in hacks
if u can't then can simply construct the args (you make a tuple) and pass to the function
but would like to see where it's not getting passed thru